wiki/Guides/Adding_Custom_Hitsounds/en.md

```diff
@@ -2,41 +2,56 @@
 
 From the thread: *[how to add custom hitsound? by: neonat](https://osu.ppy.sh/community/forums/posts/3215699)* (and modified)
 
-## Getting the custom hitsounds
+## Custom Hitsound Files
 
-You'll first need some custom hitsounds! You can either:
-- You can get them from the [Custom Hitsound Library](../Custom_Hitsound_Library/),
-- or make your own
-  - if you do make your own, try to keep the format as `.wav` - as it is best for looping.
+In order to use custom hitsound files in osu!, you'll first need some custom hitsounds! The [Custom Hitsound Library](../Custom_Hitsound_Library/) is a great resource for existing sound samples, including cymbals, drums, bells, whistles and more! Alternatively, if you can't find what you're looking for, you can make your own samples!
 
-Once you get them, move them into the song folder itself. If you don't know where your song folder is located, follow these directions:
+Hitsounds should typically be in the `.wav` or `.ogg` format, as these have no playback delay and loop correctly. `.mp3` formatted sound files have slight delays and may not accurately match up to the sounds in the music you are trying to hitsound. However, they but may still be useable for certain special hitsounds such as applause or ambient noises, where the size of `.wav` or `.ogg` files may be prohibitively large.
+
+Once you have your desired sound files, they should be moved into the song folder of the beatmap you would like to use them in. If you don't know where your song folder is located, follow these directions:
 1. Open osu!
-2. Edit
+2. Select the *Edit* option
 3. Navigate to *your beatmap* and open it
-4. Files (top-left menu, it is an off-white coloured bar at the top)
-5. Open song folder
-6. *and paste them here*
+4. Click *File* (the left-most option in an off-white coloured bar at the top)
+5. Click *Open Song Folder*
+6. *Paste your files here*
+
+Once the sound files have been placed inside the beatmap's Song Folder, they must be named appropriately in order for osu! to recongnise them as *hitsounds*.
+
+Three basic categories of hitsound, refered to as *samplesets*, exist in osu!: Normal (N), Soft (S), and Drum (D). Each sampleset can be further subdivided into different sounds. The most common of these are the "hitnormal", "hitclap", "hitwhistle" and "hitfinish" hitsounds, although some more specialized sounds exist, such as those that play during a slider ("sliderslide", "slidertick") or spinner ("spinnerspin").
```

Comment:
`categories of hitsounds, referred to` needs an s and another r in refered

Reply:
This sentence is really long. Consider splitting it up like so: `The most common of these are the "hitnormal", "hitclap", "hitwhistle" and "hitfinish" hitsounds. More specialized sounds also exist, such as those that play during a slider ("sliderslide", "slidertick") or a spinner ("spinnerspin").`

Reply:
Agreed, split it up